src/components/common/TmSelectNetwork.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <SessionFrame>
+    <div class="select-network">
+      <h2 class="session-title">Select a Network</h2>
+      <div class="select-networks-list">
+        <LiSession
+          v-for="network in networks"
+          :key="network.title"
+          icon="language"
+          :title="network.title"
+          :route="{ name: 'create', params: { network: network.id } }"
+        />
+      </div>
+    </div>
+  </SessionFrame>
+</template>
+
+<script>
+import LiSession from "common/TmLiSession"
+import SessionFrame from "common/SessionFrame"
+import gql from "graphql-tag"
+
+export default {
+  name: `select-network`,
+  components: {
+    SessionFrame,
+    LiSession
+  },
+  data: () => ({
+    networks: []
```

Comment:
can you sort the networks by putting the non-testnets first. And the `default` one on the top?

Reply:
It seems we need to reimplement the `default` field. Got somehow lost along the way

Reply:
this logic is duplicated on the networks page. let's try to re-use.